conversant/prompt_chatbot.py

```diff
@@ -406,7 +428,7 @@ def configure_client(self, client_config: Dict = {}) -> None:
         if not hasattr(self, "client_config"):
             self.client_config = {
                 "model": "xlarge",
-                "max_tokens": 100,
+                "max_tokens": 200,
```

Comment:
This change is the likely culprit for causing `test_update_max_context_warn` to fail - since `configure_client` is part of the initialization of a `mock_prompt_chatbot`